samples/csharp/v1.4-All-Samples.sln

```diff
@@ -29,12 +29,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SpikeDetection.Solution", "
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "PowerAnomalyDetection.Solution", "PowerAnomalyDetection.Solution", "{417CA47F-36DE-4F6E-B53D-330D2B373ECF}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CreditCardFraudDetection.Common", "getting-started\BinaryClassification_CreditCardFraudDetection\CreditCardFraudDetection.Common\CreditCardFraudDetection.Common.csproj", "{8FB158D4-737F-4C15-8650-A6F85810062D}"
```

Comment:
(Pinned to unrelated code to allow for threaded discussion)

@luisquintanilla  @briacht , what's the purpose of keeping these "v1.3-All-Samples.sln" and "v1.4-All-Samples.sln"?? It looks to me they were kept by @CESARDELATORRE , for some reason, to _separate_ samples that were using ML.NET 1.3.0 and 1.4.0?? Now most of the samples are using 1.5.0 anyway through these properties:

https://github.com/dotnet/machinelearning-samples/blob/522fbdba3a865589cc3e17aef7daa9554cd7aeab/samples/Directory.Build.props#L4-L5

So maybe it's better to simply remove these solutions?

Reply:
Not sure about this one, it's probably safe to remove. We want to keep samples up to date